lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -59,7 +61,24 @@ export class RefreshTokenClient extends BaseClient {
             undefined,
             true,
             request.forceCache
-        );
+        ).then((result: AuthenticationResult) => {
+            if (response.body?.refresh_token) {
+                atsMeasurement?.endMeasurement({
+                    success: true,
+                    refreshTokenSize: response?.body?.refresh_token.length || 0
```

Comment:
Should this be:
```
response?.body?.refresh_token?.length || undefined
```

Reply:
FYI that will turn length 0 into undefined. I'm assuming that's not what we want? Should verify the type of length first.

Reply:
If `response.body?.refresh_token` is truthy, the length will never be 0 (i.e. 0 is falsey). I don't understand why the if statement is needed here?

Reply:
Are we wanting to differentiate between no having a refresh token and a refresh token of length 0 (which I don't think is a scenario that can happen, and even if so, should be treated the same semantically imo.

Reply:
```suggestion
                    refreshTokenSize: response?.body?.refresh_token ? response?.body?.refresh_token.length : undefined
```

Reply:
And then remove the if statement and just have one endMeasurement call.

Reply:
This sounds feasible, just the distinction between no RT vs size 0 RT would then be treated similarly .